### PR TITLE
Add sandworm attack sound

### PR DESCRIPTION
**Pull request by penev92**
Removes a few unused lines of YAML and adds the worm sound.
May be worth delaying the player notifications by a second; it would probably sound better.

This is also the simplest thing on my Next+1 feature wishlist.

### CODE REVIEW

**Comment by Mailaender** [comment]
This should use the existing https://github.com/OpenRA/OpenRA/wiki/Traits#announceonkill instead.

**Comment by penev92** [comment]
I will look into that (see if it can work in this case), but I think that would be a pointless abstraction. The sandworm already doesn't do any normal killing.

**Comment by penev92** [comment]
I tried this but it won't work because of [this check](https://github.com/OpenRA/OpenRA/blob/0569eb640f7cd87e492d3f8dfe13cd3bc3a3dc23/OpenRA.Game/Sound/Sound.cs#L370) and I am not comfortable with making the worm selectable because of this.
Other solutions might include even further changes to `AnnounceOnKill` and other hacks, but I don't consider those nice solutions.

**Comment by abcdefg30** [comment]
> I tried this but it won't work because of this check and I am not comfortable with making the worm selectable because of this.

You could work around this issue by using
```
Selectable:
  Selectable: false
```
See https://github.com/OpenRA/OpenRA/wiki/Traits#selectable.

File: OpenRA.Mods.D2k/Activities/SwallowActor.cs [hunk 1]
```diff
@@ -89,21 +89,26 @@ bool WormAttack(Actor worm)
...
+			});
 
 			return true;
 		}
 
-		void PlayAttackAnimation(Actor self)
+		void PlayAttack(Actor self)
 		{
 			renderUnit.PlayCustomAnim(self, "mouth");
+			Sound.Play(swallow.Info.WormAttackSound, self.CenterPosition);
```
Comment (Mailaender) [inline OpenRA.Mods.D2k/Activities/SwallowActor.cs]: A `Sound.PlayThen` that would `NotifyPlayer` afterwards would be useful here for polishing.

**Comment by penev92** [comment]
@abcdefg30 I consider that an unsightly hack. I'd rather wait for a `Voiced` trait instead.

So I think what I want is an opinion whether to leave it as it is (+ @Mailaender's comment), or wait for the `Voiced` trait and use that.

**Comment by Mailaender** [comment]
#8012 seems a bit stale and far from ready to integrate. Also `AnnounceOnKill` will never be able to coordinate with the notification like this could and overlap. I am fine if we go this route. My initial comment was meant as a proposal, not a demand.

**Comment by penev92** [comment]
Added a delay on the "WormAttack" notification so it doesn't overlap with the worm attack sound. It also sounds much more believable now that is a second late.

**Comment by abcdefg30** [comment]
Looks good to me :+1: / :white_check_mark:

**Comment by obrakmann** [comment]
Code looks fine, and the effect in-game is working nicely. :+1:

**Comment by Mailaender** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/5a10ff1763b03d111b3d2d990adb9f59c6989942...7f61853a0dd7899a5f211557c407f20a02c0a80b) (thought you forgot)

**Pull request merged**